torch/_dynamo/variables/nn_module.py

```diff
@@ -567,7 +567,12 @@ def call_method(
                 return variables.ListIteratorVariable(
                     items, mutable_local=MutableLocal(), **options
                 )
-
+            elif isinstance(method, staticmethod):
+                return tx.inline_user_function_return(
+                    variables.UserFunctionVariable(method.__func__, **options),
```

Comment:
is this also part of the changes needed to "unspecialize" the module or is it something else?

Reply:
this is more of a 'missing feature of UnspecializedNNModule that is now being exercised since we are directing more traffic to it'.  I expect there are more elif cases missing here that we'll find by testing more models.  But i also claim that UnspecializedNNModule is implicitly supposed to work for all cases, since any time we mutate an NNModule we will fall back to Unspecialized.